### PR TITLE
Start to stand up a new parallel Cloud Run deployment.

### PR DESCRIPTION
**Pull request by mattmoor**
This change start to take advantage of our new common infrastructure repo that codifies a number of best practices: https://github.com/chainguard-dev/terraform-infra-common

### CODE REVIEW

File: .github/workflows/cloud-run.yaml [hunk 1]
```diff
@@ -65,3 +65,16 @@ jobs:
       with:
         service: 'academy'
         image: '${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/${{ env.REPOSITORY }}/${{ env.SERVICE }}:${{ github.sha }}'
+
+    # Attempt to deploy the terraform configuration
+    - uses: hashicorp/setup-terraform@a1502cd9e758c50496cc9ac5308c4843bcd56d36 # v2.0.0
+      with:
+        terraform_version: 1.6
+    - name: Terraform fmt
```
Comment (imjasonh) [inline .github/workflows/cloud-run.yaml:73]: Name is wrong, you can just remove it

**Review by imjasonh — APPROVED** [review]

File: .github/workflows/cloud-run.yaml (hunk 1 above)
Comment (mattmoor) [reply .github/workflows/cloud-run.yaml:73]: whoops, bad copy/paste

File: iac/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name = "chainguard-academy"
```
Comment (mattmoor) [inline iac/terraform.tfvars:1]: I changed this to avoid conflicting with the current academy deployment

**Comment by jamonation** [comment]
Got this working, stood up a parallel environment, woo!

**Review by jamonation — APPROVED** [review]

**Pull request merged**